cmake/projects/ZMQPP/hunter.cmake

```diff
@@ -9,11 +9,11 @@ hunter_add_version(
   PACKAGE_NAME
   ZMQPP
   VERSION
-  "4.1.2"
+  "4.1.2-p3"
```

Comment:
Please add new entry instead of substituting old:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package